server.js

```diff
@@ -13,9 +21,31 @@ const app = express();
 const PORT = process.env.PORT || 4000;
 
 app.set("view engine", "ejs");
-app.get('/', function (req, res) {
-    res.send('Welcome');
-  });
+
+// connect to the MongoDB with mongoose
+require('./config/database');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
require("./config/database");
```